src/routes/charger-routes.js

```diff
@@ -18,24 +20,30 @@ const fileFilter = (req, file, cb) => {
   }
 };
 
-const upload = multer({
-  storage: storage,
-  limits: { fileSize: 100000, files: 1 },
-  fileFilter,
-  rename: function(fieldname, filename) {
-    return filename.replace(/\W+/g, '-').toLowerCase();
-  },
+const upload = multer({ 
+    storage: storage,
+    limits: { fileSize: 10_000_000, files: 1 },
+    fileFilter,
+    rename: function (fieldname, filename) {
+        return filename.replace(/\W+/g, '-').toLowerCase();}
 });
 
 const chargerRouter = express.Router();
 
-chargerRouter.get('/search', searchChargersLocation);
```

Comment:
As above, is there a plan to handle the search route?

Reply:
I must have deleted this by mistake sorry hahaha. Will add it back in